style.css

```diff
@@ -486,7 +528,7 @@ button:active {
     background-image: url("images/header-bg.png");
     height: auto;
     background-repeat: no-repeat;
-    background-size: cover;
+    background-size: 100%;
     justify-content: center;
     padding: 50px 50px;
   }
```

Comment:
This section, that contains the headline looks too stretched in your page
This is your page
<img width="1679" alt="Screenshot 2023-01-17 at 14 55 52" src="https://user-images.githubusercontent.com/48546992/212892917-47bd4fa5-bf1e-4e6a-a5db-0f28e78c1cf2.png">

This is the figma design
<img width="846" alt="Screenshot 2023-01-17 at 14 49 37" src="https://user-images.githubusercontent.com/48546992/212893005-35610b95-d5bb-4b28-bbbc-10937ffddd15.png">
Note how different they are, yours looks more like a `landscape orientation` while the template is more of `portrait`
Note that you are required to "stick to the design as much as possible (e.g., font, colors, images, text, margins)" as per the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md)